web/src/marketing/SurveyPage.tsx

```diff
@@ -192,7 +192,7 @@ const TweetFeedback: React.FunctionComponent<TweetFeedbackProps> = ({ feedback,
         return (
             <>
                 <p className="mt-2">
-                    One more favor, could you share your feedback on Twitter? We'd really appreciate it!
+                    Want Sourcegraph swag? Post a tweet with your feedback and we'll send some over to you. 
```

Comment:
"Want some Sourcegraph"? Not sure about this so didn't add to suggestion. Watch for trailing spaces :) 

If we add the handle, we might get people to reach out organically (or remember to tweet us in the future) who wouldn't be comfortable "click-to-tweeting".

```suggestion
                    Want Sourcegraph swag? Share a tweet with your feedback to @srcgraph and we'll send some over to you.
```